learning-pathways/dev_tools_training.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: learning-pathway
+title: Tool development for a nice & shiny subdomain
+type: admin-dev-community
```

Comment:
```suggestion
type: admin-dev
```

this just controls where in the LP list this appears, do this for now so it will appear not in science but "For Teachers, Developers and System Administrators" 

..and we can look into how to better organize things on this page separately, now that we are getting more and more LPs :)